setup.py

```diff
@@ -36,10 +36,10 @@
                         'Markdown>=2.2.0',
                         'PyYAML>=3.10',
                         ],
-      requires=['sh (>=1.0)',
-                'Jinja2 (>=2.6)',
-                'Markdown (>=2.2.0)',
-                'PyYAML (>=3.10)',
+      requires=['sh (>=1.1)',
+                'Jinja2 (>=2.8)',
+                'Markdown (>=2.6.0)',
+                'PyYAML (>=3.12)',
```

Comment:
Just out of curiousity, is there a specific reason these requirements need to be bumped that I'm missing?